Scripts/extras/package_downloader.py

```diff
@@ -31,7 +33,9 @@
     from botocore.exceptions import BotoCoreError
     from botocore.exceptions import ClientError
     _aws_s3_available = True
+    possible_download_errors = (ClientError, BotoCoreError, ssl.SSLError, URLError, OSError)
```

Comment:
Nitpick: Could make this just `possible_download_errors = possible_download_errors + (ClientError, BotoCoreError)` in case you ever need to update the initial tuple

Reply:
sure!